conda_forge_tick/migrators/cross_compile.py

```diff
@@ -0,0 +1,74 @@
+import os
+import re
+import tempfile
+import subprocess
+import typing
+from typing import Any
+import logging
+
+from rever.tools import replace_in_file
+
+from conda_forge_tick.xonsh_utils import indir
+from conda_forge_tick.utils import eval_cmd
+from conda_forge_tick.recipe_parser import CondaMetaYAML
+from conda_forge_tick.migrators.core import (
+    MiniMigrator,
+    _get_source_code,
+)
+
+LOGGER = logging.getLogger("conda_forge_tick.migrators.cross_compile")
+
+
+class UpdateConfigSubGuessMigrator(MiniMigrator):
+    post_migration = True
+
+    def filter(self, attrs: "AttrsTypedDict", not_bad_str_start: str = "") -> bool:
+        build_reqs = attrs.get("meta_yaml", {}).get("requirements").get("build")
```

Comment:
```suggestion
        build_reqs = attrs.get("requirements"{}).get("build", set())
```
We expose the requirements top level for ease of use.